packages/apps/tools/TESTPLAN.md

```diff
@@ -0,0 +1,83 @@
+
+#### Scenario 1: On first page load the user can see all elements: Header placed at the top of the page, Side-menu that is initially opened with all available menu options, Right Drawer that is initially closed and opens on clicking the icon, and the main info content is displayed on the page
+- **Objective:** To check if a user can see all the elements on the home page and interact with them
+- **Preconditions:** Page is loaded at root level without any path-names (e.q. faucet/new)
+- **Inputs:** /
+- **Expected Outcome:** Main content of the page is loaded correctly with all elements, the user can open/close accordion, click on links and is redirected to the correct location. The main layout elements are present as well: Header has the menu items, theme toggle and network select dropdown; Sidemenu is opened and has all visible options, the user can open and close it, when link icon is clicked the links are visible as menu items; Right drawer is closed and can be opened by clicking on the info icon.
+
+
+### 3.2 [Faucet / Create and fund new accounts]
+
+#### Scenario 1: Create single key k:account and successfully fund 100 coins
+- **Objective:** User can create an account with a valid public key and fund 100 coins to it.
+- **Preconditions:** User has a public key already created.
+- **Inputs:** User enters a valid key and adds to the list on clicking '+' next to the input.
+- **Expected Outcome:** The user enters a valid public key and the account name input is automatically filled with a k:account. The user clicks on the button and after a successful creation and funding, sees a notification that the transfer was successful to the given account.
+
+#### Scenario 2: Get error on entering invalid public key (length validation)
+- **Objective:** User cannot create an account with an invalid public key and fund 100 coins to it.
+- **Preconditions:** /
+- **Inputs:** User enters an invalid key and adds to the list on clicking '+' next to the input.
+- **Expected Outcome:** The user enters an invalid public key and the input shows error status. If the user clicks on the submit button sees a error text to provide valid public key(s) and cannot fund the given account.
+
+#### Scenario 3: Create multi key w:account and successfully fund 100 coins
+- **Objective:** User can create an account with multiple valid public keys and fund 100 coins to it.
+- **Preconditions:** User has two or more public keys already created.
+- **Inputs:** User enters valid keys and adds them to the list on clicking '+' next to the input.
+- **Expected Outcome:** The user enters multiple valid public keys and the account name input is automatically filled with a w:account. The user clicks on the button and after a successful creation and funding, sees a notification that the transfer was successful to the given account.
+
+
+
+### 3.3 [Faucet / Fund existing accounts]
+
+#### Scenario 1: Successfully fund 100 coins to an existing account
```

Comment:
What if we flip the scenario around? They try to fund 100 KDA to an account that does not yet exist?

Reply:
I will add this scenario as well ✅